source/read-write-concern/read-write-concern.rst

```diff
@@ -301,11 +301,13 @@ Other commands that write
 
 Command helper methods for commands that write, other than those discussed above,
 MAY accept a write concern or write concern options in their parameter list.
+If the helper accepts a write concern, the driver MUST error if the selected server's MaxWireVersion < 5 and a
+write concern has explicitly been specified.
```

Comment:
Does the server return an error if writeConcern is sent to older servers? If so, the driver doesn't necessarily have to do any validation. If the server silently ignores the field then the driver MUST raise an error.